Competition2017/tests/competition/subsystems/drive/DriveTestBase.java

```diff
@@ -27,4 +28,8 @@ public void verifyDriveSetpoints(double left, double right) {
         assertEquals(left, ((MockCANTalon)drive.leftDrive).getSetpoint(), 0.001);
         assertEquals(right, ((MockCANTalon)drive.rightDrive).getSetpoint(), 0.001);
     }
+    
+    public void setMockGyro(double heading) {
```

Comment:
How about `setRobotHeading`? The fact that it's a mock gyro isn't important to the caller, given that we only use gyros.